### PR TITLE
modifying the README to include 'Allow access to file URLs' note

### PR DESCRIPTION
**Pull request by mabdalmoniem**
In this PR I've modified the README to account for the permission of Chrome extension: Allow access to file URLs'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,16 +10,37 @@ alpinejs-devtools is a simple extension to help you debug [Alpine.js](https://gi
 
 ## Installation
 
-- [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
-- [Get the Firefox Extension](https://addons.mozilla.org/firefox/addon/alpinejs-devtools/)
+-   [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
+-   [Get the Firefox Extension](https://addons.mozilla.org/firefox/addon/alpinejs-devtools/)
+
+If you are using Google Chrome, make sure that you are have gave the extension a permission to `Access to File URLs`, you can do this by following these steps:
```
Comment (HugoDF) [inline README.md:16]: If you are using Google Chrome, Alpine Devtools won't work with local files unless you configure the extension to be allowed `Access to File URLs`

File: README.md [hunk 2]
```diff
@@ -10,16 +10,37 @@ alpinejs-devtools is a simple extension to help you debug [Alpine.js](https://gi
...
+
+1. Open chrome settings
+2. Go to `extensions` tab
+3. Choose `Alpine.js devtools` and press details
+4. enable `Allow access to file URLs`
```
Comment (HugoDF) [inline README.md:21]: Probably should be "Enable"

File: README.md [hunk 3]
```diff
@@ -10,16 +10,37 @@ alpinejs-devtools is a simple extension to help you debug [Alpine.js](https://gi
...
+
+Check the following screenshot with it enabled
+
+<p align="center">
+    <img src="docs/alpine-devtools-chrome-permission.png" alt="Screenshot of Alpine.js DevTools">
```
Comment (HugoDF) [inline README.md:26]: can we get rid of the HTML and use markdown image markup?

File: README.md [hunk 4]
```diff
@@ -10,16 +10,37 @@ alpinejs-devtools is a simple extension to help you debug [Alpine.js](https://gi
...
+</p>
+
+You can add `name` attribute to components to easily identify them while debugging, example:
```
Comment (HugoDF) [inline README.md:29]: Maybe:

To help with inspection, component "names" are be computed from the following attributes (if present and in order of precedence): `id`, `name`, `x-data` function name (if a function is used), tag name

File: README.md [hunk 5]
```diff
@@ -10,16 +10,37 @@ alpinejs-devtools is a simple extension to help you debug [Alpine.js](https://gi
 
 ## Installation
 
-- [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
-- [Get the Firefox Extension](https://addons.mozilla.org/firefox/addon/alpinejs-devtools/)
+-   [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
```
Comment (HugoDF) [inline README.md:13]: Any reason for the extra whitespace?

**Comment by HugoDF** [comment]
By the way, this closes #47

File: README.md (hunk 5 above)
Comment (mabdalmoniem) [reply README.md:13]: Prettier adds the extra space

Comment (HugoDF) [reply README.md:11]: Would be better without but maybe we should have linting/formatting in place 🤷🏼‍♀️

**Pull request merged**